### PR TITLE
Move afv to afvl

### LINKED ISSUE #3702: `afv` command changes
- Move `afv` to `afvl`
- Add TABLE mode to `afvlt`
- Add SHORT mode to `afvl`
- Add LONG mode for `afvll`

Print more details when available - e.g. from loclists or CFA in LONG and TABLE modes

### PR DESCRIPTION
**Pull request by b1llow**
**Your checklist for this pull request**
- [ ] I've read the [guidelines for contributing](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md) to this repository
- [ ] I made sure to follow the project's [coding style](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md#code-style)
- [ ] I've documented or updated the documentation of every function and struct this PR changes. If not so I've explained why.
- [ ] I've added tests that prove my fix is effective or that my feature works (if possible)
- [ ] I've updated the [rizin book](https://github.com/rizinorg/book) with the relevant information (if needed)

**Detailed description**

- Move `afv` to `afvl`
- Add TABLE mode to `afvlt`
- Add SHORT mode to `afvl`
- Add LONG mode for `afvll`

Print more details when available - e.g. from loclists or CFA in LONG and TABLE modes

**Test plan**

...

**Closing issues**

closes #3702

### CODE REVIEW

File: test/db/analysis/vars [hunk 1]
```diff
@@ -892,6 +892,19 @@ EXPECT=<<EOF
 var va_list ap @ stack + 0x4
 arg const char *fmt @ a4
 var int ans @ ...
+is_arg name type         constraints origin addr                                                                                                                        
+------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+false  ap   va_list                  DWARF  loclist: [
```
Comment (XVilka) [inline test/db/analysis/vars]: Maybe put them all in one line?

File: test/db/analysis/vars [hunk 2]
```diff
@@ -1098,150 +1268,3 @@ EXPECT=<<EOF
 \           0x8000054e      ret
 EOF
 RUN
-
-NAME=Function va_args arguments from DWARF
```
Comment (XVilka) [inline test/db/analysis/vars:1102]: Why removing this one?

**Review by XVilka — CHANGES_REQUESTED** [review]
Overall, it is good. Some tests are broken, though.

File: test/db/analysis/vars (hunk 2 above)
Comment (b1llow) [reply test/db/analysis/vars:1102]:
```
NAME=Function va_args arguments from DWARF
FILE=bins/elf/float_ex1/float_ex1_hightec
CMDS=<<EOF
aaa
(sig; afv; afs; pd 1; echo "")
s dbg.printf
.(sig)
s dbg.prout
.(sig)
s dbg._fpadd_parts
.(sig)
s dbg.strnlen_s
.(sig)
s dbg._Wctomb
.(sig)
s dbg._Mbtowc
.(sig)
s dbg.feraiseexcept
.(sig)
s dbg._Atrealloc
.(sig)
EOF
```

```
NAME=Function va_args arguments from DWARF
FILE=bins/elf/float_ex1/float_ex1_hightec
CMDS=<<EOF
aaa
(sig; afvl; afvlt; afs; pd 1; echo "---------")
s dbg.printf
.(sig)
s dbg.prout
.(sig)
s dbg._fpadd_parts
.(sig)
s dbg.strnlen_s
.(sig)
s dbg._Wctomb
.(sig)
s dbg._Mbtowc
.(sig)
s dbg.feraiseexcept
.(sig)
s dbg._Atrealloc
.(sig)
s dbg.fn1
.(sig)
EOF
```

The first test was deleted, the second was already there, and I thought they were almost identical and the second test tested more content, so I deleted the first one.

File: test/db/analysis/vars (hunk 1 above)
Comment (b1llow) [reply test/db/analysis/vars]: done

**Review by wargio — APPROVED** [review]

**Review by XVilka — COMMENTED** [review]
Not sure why this unrelated test fails:
```
[XX] db/archos/linux-x64/dbg_dmh dmh allocated
RZ_NOPLUGINS=1 /usr/bin/rizin -escr.utf8=0 -escr.color=0 -escr.interactive=0 -eflirt.sigdb.load.system=false -eflirt.sigdb.load.home=false -N -d -Qc 'dcu main
dmh | tail -n 2
echo ----
7dso
dmh | tail -n 3
' bins/elf/simple_malloc_x86_64
-- stdout
--- expected
+++ actual
@@ -1,6 +1,6 @@
-Chunk(status=allocated,size=0x11c10)
-Chunk(status=free,size=0xf160)
+Chunk(status=allocated,size=0x12010)
+Chunk(status=free,size=0xed60)
 ----
-Chunk(status=allocated,size=0x11c10)
+Chunk(status=allocated,size=0x12010)
 Chunk(status=allocated,size=0x20)
-Chunk(status=free,size=0xf140)
+Chunk(status=free,size=0xed40)

-- stderr
Process with PID 14405 started...
WARNING: Neither hash nor gnu_hash exist. Falling back to heuristics for deducing the number of dynamic symbols...
WARNING: Neither hash nor gnu_hash exist. Falling back to heuristics for deducing the number of dynamic symbols...
WARNING: Neither hash nor gnu_hash exist. Falling back to heuristics for deducing the number of dynamic symbols...
Continue until 0x5569e900070a
hit breakpoint at: 0x5569e900070a
hit breakpoint at: 0x5569e9000723
```
Updated GLIBC perhaps?

**Review by XVilka — APPROVED** [review]
LGTM

**Comment by b1llow** [comment]
> Not sure why this unrelated test fails:
> 
> ```
> [XX] db/archos/linux-x64/dbg_dmh dmh allocated
> RZ_NOPLUGINS=1 /usr/bin/rizin -escr.utf8=0 -escr.color=0 -escr.interactive=0 -eflirt.sigdb.load.system=false -eflirt.sigdb.load.home=false -N -d -Qc 'dcu main
> dmh | tail -n 2
> echo ----
> 7dso
> dmh | tail -n 3
> ' bins/elf/simple_malloc_x86_64
> -- stdout
> --- expected
> +++ actual
> @@ -1,6 +1,6 @@
> -Chunk(status=allocated,size=0x11c10)
> -Chunk(status=free,size=0xf160)
> +Chunk(status=allocated,size=0x12010)
> +Chunk(status=free,size=0xed60)
>  ----
> -Chunk(status=allocated,size=0x11c10)
> +Chunk(status=allocated,size=0x12010)
>  Chunk(status=allocated,size=0x20)
> -Chunk(status=free,size=0xf140)
> +Chunk(status=free,size=0xed40)
> 
> -- stderr
> Process with PID 14405 started...
> WARNING: Neither hash nor gnu_hash exist. Falling back to heuristics for deducing the number of dynamic symbols...
> WARNING: Neither hash nor gnu_hash exist. Falling back to heuristics for deducing the number of dynamic symbols...
> WARNING: Neither hash nor gnu_hash exist. Falling back to heuristics for deducing the number of dynamic symbols...
> Continue until 0x5569e900070a
> hit breakpoint at: 0x5569e900070a
> hit breakpoint at: 0x5569e9000723
> ```
> 
> Updated GLIBC perhaps?

I'm feeling weird, too. My local environment passes this test.

**Comment by b1llow** [comment]
The same problem occurs in the dev branch.

https://github.com/rizinorg/rizin/actions/runs/6648566447/job/18154656280

**Comment by XVilka** [comment]
Ok, let's merge then and update the test separately, it just means the underlying Glibc has changed on github

**Pull request merged**